src/pollution_model/pollution_tables.py

```diff
@@ -19,48 +19,54 @@ class _BaseMedusa2ModelOutput(Base):
 
     Attributes
     ----------
-    __tablename__ : str
-       Name of the database table.
     scenario_id : int
-       The rainfall event scenario ID this feature was processed with, makes up part of the primary key.
-    spatial_feature_id: int
-       The id of the spatial feature this row is associated with, makes up part of the primary key.
+        The rainfall event scenario ID this feature was processed with, makes up part of the primary key.
     surface_type : str
-       The type of surface this is. Converted to string from Enum.
+        The type of surface this is. Converted to string from Enum.
     total_suspended_solids : float
-       Total suspended solids of the surface.
+        Total suspended solids of the surface.
     total_copper : float
-       Total copper of the surface.
+        Total copper of the surface.
     total_zinc: float
-       Total zinc of the surface.
+        Total zinc of the surface.
     dissolved_copper float
-       Total dissolved copper of the surface.
+        Total dissolved copper of the surface.
     dissolved_zinc: float
-       Total dissolved zinc of the surface.
+        Total dissolved zinc of the surface.
     """
 
     __abstract__ = True  # This is an abstract base class
 
     @property
     @abc.abstractmethod
     def __tablename__(self) -> str:
-        """This must be overridden in the child class. e.g. __tablename__ = "medusa2_model_output_buildings."""
-        raise NotImplementedError("__tablename__ must be overidden in the child class."
-                                  "e.g. __tablename__ = \"medusa2_model_output_buildings.\"")
+        """
+        Name of the database table.
+        This must be overridden in the child class. e.g. __tablename__ = "medusa2_model_output_buildings."
+        """

```

Comment:
Include the `Returns` section and maybe the `Raises` section in the method docstrings.